<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherModuleWidget</class>
 <widget class="QWidget" name="TeacherModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>881</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>当前课程：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_course">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0, 255, 127, 200);
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:rgba(255,255,255,30);
font: 12pt &quot;楷体&quot;;
color:rgb(0,0,0);
padding:2px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_change_course">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgba(0, 255, 127, 200);
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:rgba(255,255,255,30);
font: 12pt &quot;楷体&quot;;
color:rgb(0,0,0);
padding:3px;</string>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string notr="true">成绩录入</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>851</width>
          <height>401</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>学生搜索： </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edit_search">
             <property name="styleSheet">
              <string notr="true">background-color:rgba(255, 255, 255, 200);
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:rgba(0, 255, 127, 200);
font: 11pt &quot;楷体&quot;;
color:rgb(0,0,0);
padding:2px;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_search">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgba(0, 255, 127, 200);
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:rgba(255,255,255,30);
font: 12pt &quot;楷体&quot;;
color:rgb(0,0,0);
padding:3px;</string>
             </property>
             <property name="text">
              <string>查询</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="button_readFile">
             <property name="styleSheet">
              <string notr="true">background-color:rgba(0, 255, 127, 200);
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:rgba(255,255,255,30);
font: 12pt &quot;楷体&quot;;
color:rgb(0,0,0);
padding:3px;</string>
             </property>
             <property name="text">
              <string>文件录入</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_save_score">
             <property name="styleSheet">
              <string notr="true">background-color:rgba(0, 255, 127, 200);
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:rgba(255,255,255,30);
font: 12pt &quot;楷体&quot;;
color:rgb(0,0,0);
padding:3px;</string>
             </property>
             <property name="text">
              <string>保存更改</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="tableView_set_score">
           <property name="styleSheet">
            <string notr="true">background-color: rgba(41, 139, 201, 150);
border-color: rgba(41, 139, 201, 150);
font: 12pt;
color: black;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string notr="true">成绩查询</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>11</y>
          <width>851</width>
          <height>401</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>选择班级：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_classNo">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgba(0, 255, 127, 200);
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:rgba(255,255,255,30);
font: 12pt &quot;楷体&quot;;
color:rgb(0,0,0);
padding:2px;</string>
             </property>
             <item>
              <property name="text">
               <string>所有班级</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>搜索学生：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edit_student">
             <property name="styleSheet">
              <string notr="true">background-color:rgba(255, 255, 255, 200);
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:rgba(0, 255, 127, 200);
font: 11pt &quot;楷体&quot;;
color:rgb(0,0,0);
padding:2px;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>成绩区间：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edit_low">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgba(255, 255, 255, 200);
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:rgba(0, 255, 127, 200);
font: 11pt &quot;楷体&quot;;
color:rgb(0,0,0);
padding:2px;</string>
             </property>
             <property name="maxLength">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edit_high">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgba(255, 255, 255, 200);
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:rgba(0, 255, 127, 200);
font: 11pt &quot;楷体&quot;;
color:rgb(0,0,0);
padding:2px;</string>
             </property>
             <property name="maxLength">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="button_search_2">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgba(0, 255, 127, 200);
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:rgba(255,255,255,30);
font: 12pt &quot;楷体&quot;;
color:rgb(0,0,0);
padding:3px;</string>
             </property>
             <property name="text">
              <string>查询</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="button_analysis">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgba(0, 255, 127, 200);
border-style:outset;
border-width:2px;
border-radius:5px;
border-color:rgba(255,255,255,30);
font: 12pt &quot;楷体&quot;;
color:rgb(0,0,0);
padding:3px;</string>
             </property>
             <property name="text">
              <string>成绩分析</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="tableView_score">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(41, 139, 201, 150);
border-color: rgba(41, 139, 201, 150);
font: 12pt;
color: black;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string notr="true">统计分析</string>
       </attribute>
       <widget class="QChartView" name="chartWidget" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>851</width>
          <height>401</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(41, 139, 201, 150);
border-color: rgba(41, 139, 201, 150);
font: 12pt;
color: black;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string notr="true">课程评价</string>
       </attribute>
       <widget class="QListView" name="listView_comment">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>851</width>
          <height>401</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(41, 139, 201, 150);
border-color: rgba(41, 139, 201, 150);
font: 12pt;
color: black;
selection-background-color: rgb(0, 85, 255)</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
